src/kv/RocksDBStore.cc

```diff
@@ -899,7 +899,7 @@ void RocksDBStore::RocksDBTransactionImpl::put_bat(
 			   to_set_bl.length()));
   } else {
     rocksdb::Slice key_slice(key);
-    vector<rocksdb::Slice> value_slices(to_set_bl.buffers().size());
+    vector<rocksdb::Slice> value_slices(to_set_bl.get_num_buffers());
```

Comment:
off-topic, shall we s/list::get_num_buffers/list::size/ ? as `list` is, semantically, a list of `buffer::ptr` . so its size should be the number of elements in this list.